server/src/_gen-prisma-classes/room.ts

```diff
@@ -14,6 +15,8 @@ export class Room {
   description?: string;
 
   @ApiProperty({ type: Number })
+  @IsNotEmpty()
```

Comment:
ayaw iedit ang genfile bayot, auto generated mana. Pag create ug custom dto file unya ioverride lang na nga property para mabutangan nimo ug decorator ([How to override parent class properties](https://stackoverflow.com/questions/70434444/override-parent-class-properties-typescript))

Reply:
ang reason nganong dili dapat iedit ang gen file kay everytime morun ang prisma generator, tanan nimong giedit mawala to, kinahanglan napod nimo imanual ug edit tanan files

Reply:
nalimot diay kog kuha sorry

Reply:
dli man mawala baby girl

Reply:
ako na ge restart og pa run naa raman